### PR TITLE
New package: gama-1.1.4

### PR DESCRIPTION
**Pull request by zen0bit**
#### Testing the changes
- I tested the changes in this PR: **briefly**

#### New package
- This new package conforms to the [package requirements](https://github.com/void-linux/void-packages/blob/master/CONTRIBUTING.md#package-requirements): **YES**

#### Local build testing
- I built this PR locally for my native architecture, x64 glibc

### CODE REVIEW

**Review by meator — CHANGES_REQUESTED** [review]
Remove all commented out variables. Apart from that, I've checked everything and I have not found any other problems. I am unsure whether the repo is GPL-3.0-only or GPL-3.0-or-later because upstream makes no mention of this, but that is a nitpick.

**Comment by zen0bit** [comment]
> Remove all commented out variables. Apart from that, I've checked everything and I have not found any other problems. I am unsure whether the repo is GPL-3.0-only or GPL-3.0-or-later because upstream makes no mention of this, but that is a nitpick.

Done
question passed upstream

**Review by zen0bit — COMMENTED** [review]
Done

File: srcpkgs/gama/template [hunk 1]
```diff
@@ -0,0 +1,12 @@
+# Template file for 'gama'
+pkgname=gama
+version=1.0.4
```
Comment (canack) [inline srcpkgs/gama/template:3]: Hi, I've released new version. And this version includes new description about license. 📄 
```suggestion
version=1.0.5
```

**Review by canack — APPROVED** [review]
Thank you for your interest in gama! 🚀

**Comment by meator** [comment]
When there is no [mention of "or later", it is GPL-3.0-only](https://www.gnu.org/licenses/identify-licenses-clearly.html) AFAIK. So the previous notice was valid and it was GPL-3.0-only. But the author changed his license making this clear. These two variants are usually differentiated in license comment headers in code itself, but the project doesn't use these.

But I'm not a license expert.

**Comment by canack** [comment]
Thank you very much for the information. I am not very knowledgeable about licenses, and I will pay more attention to this matter in my future programming processes. 👍🏻

**Comment by canack** [comment]
Hi 👋🏻 
Gama v1.1.0 has been released : https://github.com/termkit/gama/releases
May you can update the package version 🍀

**Pull request closed without merging**